### PR TITLE
Development

### PR DESCRIPTION
**Pull request by harshitsaxena586**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles/index-style.css">
+    <title>Document</title>
```
Comment (harshith-venkatesh) [inline index.html]: Title can be changes to Neutron UI and fav icon can be added for better Title view

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    <div class="container">
+        <!DOCTYPE html>
+        <html lang="en">
+
+        <head>
+            <meta charset="UTF-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Document</title>
```
Comment (harshith-venkatesh) [inline index.html]: title change to more suitable name

**Comment by harshith-venkatesh** [comment]
image
Adding a margin here would look good

**Comment by harshith-venkatesh** [comment]
Awesome Design dude , just love it

**Comment by harshith-venkatesh** [comment]
image
I really liked the way you have notified the user about non mobile responsiveness
I hope you have thought to finish it soon 
It was fun doing visual component UI review

**Comment by harshith-venkatesh** [comment]
image
I really liked the way you have notified the user about non mobile responsiveness
I hope you have thought to finish it soon
It was fun doing visual component UI review

**Comment by pruthvirajmv** [comment]
Overall the code is well written
One thing noticed is, the documentation page, few components are hidden in the side navigation. Below is the reference
image

**Comment by sakshijainn** [comment]
Hi Harshit,
I think you can change the hover color o your components list: red seems off . Also Try to have a semantic hover effect not depending on the size of the components/letter of li elements 

For Reference:
image
I think you should  add a a vertical scrolling for components , they are not moving while window is scrolling down 

Also I check make it mobile responsive too . Maybe rt now a basic boilerplate for that .

**Pull request merged**